cves/CVE-2014-9604.yml

```diff
@@ -171,10 +172,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: Mateusz "j00ru" Jurczyk and Gynvael Coldwind
```

Comment:
Is there a story behind how it was found (ie. a backstory or a bug report) or did they just stumble across it and notice it one day?

Reply:
Was it automated? Was it from a bug report?